### PR TITLE
Feat cache labels

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

To help the SDK generator generate code for offline support, 3 new labels are needed:

* `sdk.offline.model` - Refers to the local collection where offline data should be stored. No value indicates the data will not be cached
* `sdk.offline.key` - Refers to the ID of the local data
* `sdk.offline.response.key` - JSON property name that has the ID. Defaults to $id

## Test Plan

Generated updated specs:

```
docker compose exec appwrite specs version=latest mode=normal
```

## Related PRs and Issues

None

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: docs/tutorials/add-route.md [hunk 1]
```diff
@@ -77,8 +77,11 @@ App::post('/v1/account/create')
 * sdk.namespace - Refers to the route namespace.
 * sdk.method - Refers to the sdk method that needs to called.
 * sdk.description - Description of the route, using markdown format.
-* sdk.sdk.response.code - Refers to the route http response status code expected.
+* sdk.response.code - Refers to the route http response status code expected.
 * sdk.auth.response.model - Refers the route http response expected.
+* sdk.offline.model - Refers to the local collection where offline data should be stored. No value indicates the data will not be cached
+* sdk.offline.key - Refers to the ID of the local data
+* sdk.offline.response.key.id - JSON property name that has the ID. Defaults to $id
```
Comment (stnguyen90) [inline docs/tutorials/add-route.md:84]: @eldadfux, I'm open to renaming the labels

File: src/Appwrite/Specification/Format/OpenAPI3.php [hunk 2]
```diff
@@ -170,6 +170,9 @@ public function parse(): array
                     'scope' => $route->getLabel('scope', ''),
                     'platforms' => $sdkPlatforms,
                     'packaging' => $route->getLabel('sdk.packaging', false),
+                    'offline-model' => $route->getLabel('sdk.offline.model', ''),
+                    'offline-key' => $route->getLabel('sdk.offline.key', ''),
+                    'offline-response-key-id' => $route->getLabel('sdk.offline.response.key.id', '$id'),
```
Comment (stnguyen90) [inline src/Appwrite/Specification/Format/OpenAPI3.php:175]: I'm also open to renaming these keys

Comment (eldadfux) [reply src/Appwrite/Specification/Format/OpenAPI3.php:175]: `offline-response-key-id` could be `offline-response-key`

Comment (stnguyen90) [reply src/Appwrite/Specification/Format/OpenAPI3.php]: @eldadfux, I've updated the label.

**Review by eldadfux — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**